### DISCUSSION TITLE
Drag and Drop (move) Pictures into another folder

### DISCUSSION THREAD

**Discussion by FullByte** [ente-io/ente#2638] [Enhancements]
On desktop and mobile I can select an image and copy it to another album. 
It would be great for sorting images if it is possible to directly drag and drop images to other folders and move them there (not keep them in the original folder).
This is especially useful e.g. when syncing and sorting out mobile images. (a.k.a keeping the inbox clean :) )

In short:
- drag and drop, not select and choose action
- Not copy, but move images

**Comment by mnvr** [comment]
The drag and drop bit is not implemented yet, but just so you're aware, the move functionality is already present on desktop. If you open an album and select a bunch of photos in it, then on the action bar you'll see the move option

<img width="430" alt="Screenshot 2024-08-08 at 20 32 15" src="https://github.com/user-attachments/assets/2502771d-6f9a-4f3e-b56e-cfb0818c25b4">

---

(And there'll be a similar thing on mobile too)

**Reply by FullByte** [reply]
Thanks for highlighting the move option. I noticed that the move option is not available when viewing "all albums". To improve usability, I think it would help to have this option always available and don't see any logic issue with moving images from (possibly) various sources to one selected destination. This would also apply to drag-and-drop (which should work anywhere – also from “all albums”).

**Comment by adib006** [comment]
I am looking for a way to manually rearrange and relocate the positions of my albums on the dashboard grid. Currently, Ente forces a chronological or alphabetical structure. I need to be able to move specific albums down to the bottom of the page to hide them from immediate view, or position them precisely where I want without relying on renaming workarounds.

Most native gallery apps allow users to reorganize their folders freely. I understand that syncing custom layouts over end-to-end encryption takes work, but having control over our dashboard presentation would actually improve daily library management.

In short:
• Manual Reordering: Let users drag or move albums to custom grid positions.
• Archive Support: Make it easy to push less-used albums to the very bottom of the page.
• Menu Shortcut: Alternatively, add a simple "Move to bottom" option in the three-dot menu.

Thank you, devs!
